src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -5088,6 +5088,125 @@ def test_replication_mode(self, resource_group):
+        self.cmd('sig show --gallery-name {gallery} --resource-group {rg} --select Permissions', checks=[
+            self.check('sharingProfile.permissions', 'Private')
+        ])
+
+    @ResourceGroupPreparer(location='CentralUSEUAP')
+    def test_create_vm_with_community_gallery_image(self, resource_group, resource_group_location):
+        self.kwargs.update({
+            'vm': self.create_random_name('vm', 16),
+            'vm_with_community_gallery': self.create_random_name('vm_sg', 16),
+            'vm_with_community_gallery_version': self.create_random_name('vm_sgv', 16),
+            'vm_with_community_gallery_version2': self.create_random_name('vm_sgv2', 16),
+            'vmss_with_community_gallery_version': self.create_random_name('vmss', 16),
+            'gallery': self.create_random_name('gellery', 16),
+            'image': self.create_random_name('image', 16),
+            'version': '1.1.2',
+            'captured': 'managedImage1',
+            'location': resource_group_location,
+            'subId': '0b1f6471-1bf0-4dda-aec3-cb9272f09590',
+            'tenantId': '2f4a9838-26b7-47ee-be60-ccc1fdec5953',
+        })
+
+        self.cmd('sig create -g {rg} --gallery-name {gallery} --permissions Community --publisher-uri puburi --publisher-email abc@123.com --eula eula --public-name-prefix pubname')
+        self.cmd('sig share enable-community -r {gallery} -g {rg}')
+
+        self.cmd('sig image-definition create -g {rg} --gallery-name {gallery} --gallery-image-definition {image} --os-type linux -p publisher1 -f offer1 -s sku1')
+        self.cmd('vm create -g {rg} -n {vm} --image ubuntults --admin-username gallerytest --generate-ssh-keys --nsg-rule None')
+        self.cmd('vm deallocate -g {rg} -n {vm}')
+        self.cmd('vm generalize -g {rg} -n {vm}')
+
+        self.cmd('image create -g {rg} -n {captured} --source {vm}')
+        self.cmd('sig image-version create -g {rg} --gallery-name {gallery} --gallery-image-definition {image} --gallery-image-version {version} --managed-image {captured} --replica-count 1')
+        self.kwargs['public_name'] = self.cmd('sig show --gallery-name {gallery} --resource-group {rg} --select Permissions').get_output_in_json()['sharingProfile']['communityGalleryInfo']['publicNames'][0]
+
+        self.cmd('sig image-version show-community --gallery-image-definition {image} --public-gallery-name {public_name} --location {location} --gallery-image-version {version}',
+            checks=[
+                self.check('location', '{location}'),
+                self.check('name', '{version}'),
+                self.check('uniqueId', '/CommunityGalleries/{public_name}/Images/{image}/Versions/{version}')
+            ])
+
+        self.kwargs['community_gallery_image_version'] = self.cmd('sig image-version show -g {rg} --gallery-image-definition {image} --gallery-image-version {version} --gallery-name {gallery}').get_output_in_json()['id']
```

Comment:
Please use `sig image-version show-community` instead of `sig image-version show`